Makefile

```
@@ -15,15 +15,13 @@ test:
 
 .PHONY: devel-deps
 devel-deps:
-	@go get -v -u github.com/Songmu/ghch/cmd/ghch
-	@go get -v -u github.com/Songmu/goxz/cmd/goxz
 	@go get -v -u github.com/git-chglog/git-chglog/cmd/git-chglog
 	@go get -v -u github.com/golang/dep/cmd/dep
 	@go get -v -u golang.org/x/lint/golint
 	@go get -v -u github.com/haya14busa/goverage
 	@go get -v -u github.com/motemen/gobump/cmd/gobump
-	@go get -v -u github.com/tcnksm/ghr
 	@curl -sfL https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh| sh -s -- -b $(go env GOPATH)/bin
+	@curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh | BINDIR=$(go env GOPATH)/bin sh
```

I got this error 
```console
➜  tfnotify git:(dtan4/tidy-up-devel-deps) make devel-deps
...
reviewdog/reviewdog info checking GitHub for latest tag
reviewdog/reviewdog info found version: 0.9.14 for v0.9.14/Darwin/x86_64
install: /bin//reviewdog: Operation not permitted
make: *** [devel-deps] Error 71
```
It seems `$(go env GOPATH)` doesn't work expectedly. So I think It's better to use `$$` for shell `$`:
```suggestion
	@curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh | BINDIR=$$(go env GOPATH)/bin sh
```
( for reviewdog installation as well )

---

Thanks! I replaced both reviewdog and GoRelease to use `$$` https://github.com/mercari/tfnotify/pull/59/commits/afbd54abf540937b48ef3b55748c65b622371e5d